<ui version="4.0" >
 <class>Dialog</class>
 <widget class="AAdvancedEventSettings" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Advanced Event Settings</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>10</y>
     <width>281</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="sizeConstraint" >
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="topMargin" >
     <number>2</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label" >
      <property name="font" >
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet" >
       <string notr="true" >color: rgb(140, 0, 0);</string>
      </property>
      <property name="text" >
       <string>Advanced Event Settings</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet" >
       <string notr="true" >color: rgb(0, 0, 180);</string>
      </property>
      <property name="text" >
       <string>Track Reconstruction Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="TrackComboBox" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="editable" >
       <bool>false</bool>
      </property>
      <property name="insertPolicy" >
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy" >
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="duplicatesEnabled" >
       <bool>false</bool>
      </property>
      <property name="frame" >
       <bool>true</bool>
      </property>
      <property name="modelColumn" >
       <number>0</number>
      </property>
      <item>
       <property name="text" >
        <string>STrack</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet" >
       <string notr="true" >color: rgb(0, 0, 180);</string>
      </property>
      <property name="text" >
       <string>Missing ET Reconstruction Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="MisEtComboBox" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="editable" >
       <bool>false</bool>
      </property>
      <property name="insertPolicy" >
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy" >
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="duplicatesEnabled" >
       <bool>false</bool>
      </property>
      <property name="frame" >
       <bool>true</bool>
      </property>
      <property name="modelColumn" >
       <number>0</number>
      </property>
      <item>
       <property name="text" >
        <string>mMET_Final</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>mMET_RefMuon</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>mMET_Calib</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>mMET_RefFinal</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>mMET_Truth</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="font" >
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet" >
       <string notr="true" >color: rgb(0, 0, 180);</string>
      </property>
      <property name="text" >
       <string>Jet Reconstruction Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="JetComboBox" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="font" >
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground" >
       <bool>false</bool>
      </property>
      <property name="editable" >
       <bool>false</bool>
      </property>
      <property name="insertPolicy" >
       <enum>QComboBox::InsertAtBottom</enum>
      </property>
      <property name="sizeAdjustPolicy" >
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="duplicatesEnabled" >
       <bool>false</bool>
      </property>
      <property name="frame" >
       <bool>true</bool>
      </property>
      <property name="modelColumn" >
       <number>0</number>
      </property>
      <item>
       <property name="text" >
        <string>Kt4H1TowerJets</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Kt4H1TopoJets</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Cone4H1TowerJets</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Cone4H1TopoJets</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>500</width>
        <height>50</height>
       </size>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AAdvancedEventSettings</class>
   <extends>QDialog</extends>
   <header>AAdvancedEventSettings.h</header>
   <container>1</container>
   <slots>
    <slot>defineTrackType(Qstring)</slot>
    <slot>defineJetType(QString)</slot>
    <slot>defineMisEtType(Qstring)</slot>
    <slot>defineTrackType(QString)</slot>
    <slot>defineMisEtType(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>230</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>230</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>JetComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>defineJetType(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel" >
     <x>155</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MisEtComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>defineMisEtType(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel" >
     <x>155</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TrackComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>defineTrackType(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel" >
     <x>155</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
